### ISSUE TITLE
Creating a new realm or selecting an existing one requires two clicks

### ISSUE THREAD

**Issue by inad9300** [keycloak/keycloak#15468]
### Area

admin/ui

### Describe the bug

On the menu shown below, whenever I click on "Create Realm" or on any realm to navigate to it, it does nothing. It is only after a second click that the navigation occurs.

![image](https://user-images.githubusercontent.com/3440331/201355036-fbfa8fad-04c8-4277-ab41-c2af7c6e0039.png)

### Version

quay.io/keycloak/keycloak:20.0.1

**Comment by ssilvert** [comment]
Thanks for reporting @inad9300.  This is a duplicate of https://github.com/keycloak/keycloak-ui/issues/3742
